packages/cli/src/validators/no-xss.validator.ts

```diff
@@ -4,7 +4,13 @@ import { registerDecorator, ValidatorConstraint } from 'class-validator';
 
 @ValidatorConstraint({ name: 'NoXss', async: false })
 class NoXssConstraint implements ValidatorConstraintInterface {
-	validate(value: string) {
+	validate(value: string | string[]) {
+		if (Array.isArray(value)) return value.every((item) => this.validateItem(item));
+
+		return this.validateItem(value);
+	}
```

Comment:
Would be nicer to keep the `NoXss` validator as a string only and use the [`each`](https://www.npmjs.com/package/class-validator#validating-arrays) option in `class-validator` to validate arrays. That way it behaves the same way as the built-in validators in `class-validator`.

Also we should make sure that the `value` we receive is of the the expected type (i.e. `string`). If it's not, we should then mark the validation as failed for security (safe defaults)

Reply:
Thank you! I was not familiar with this option.